routes/api_worldometers.js

```diff
@@ -71,6 +71,14 @@ router.get('/states', async (req, res) => {
 	res.send(states);
 });
 
+router.get('/states/:query', async (req, res) => {
+	const { query } = req.params;
+	const states = JSON.parse(await redis.get(keys.states));
+	const stateData = states.filter(st => st.state === query);
+	const state = stateData[0] || {};
```

Comment:
please use
`const state = states.find(st => st.state === query)` to make it shorter

Reply:
Done